pkg/factory/risingwave_object_factory.go

```diff
@@ -502,6 +502,26 @@ func (f *RisingWaveObjectFactory) getDataDirectory() string {
 	return path.Join(rootPath, stateStore.DataDirectory)
 }
 
+func (f *RisingWaveObjectFactory) envsForLicense() []corev1.EnvVar {
+	license := f.risingwave.Spec.License
+	if license != nil && license.SecretName != "" {
+		return []corev1.EnvVar{
+			{
+				Name: envs.RWLicense,
+				ValueFrom: &corev1.EnvVarSource{
+					SecretKeyRef: &corev1.SecretKeySelector{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: license.SecretName,
+						},
+						Key: corev1.LicenseKey,
+					},
```

Comment:
There isn't such symbol in `corev1` package. I think we need to define our own.

Reply:
😄 I found that I modified the code in `vendor/`. Let me fix it.